src/types/dense.jl

```diff
@@ -113,4 +113,9 @@ end
 
 save(io::IO, data::DenseTaggedImage) where {IO <: Union{IOStream, Stream}} = write(io, data)
 save(io::IO, data) where {IO <: Union{IOStream, Stream}} = save(io, DenseTaggedImage(data))
-save(filepath::String, data) = save(Stream(format"TIFF", open(filepath, "w"), filepath), data)
+function save(filepath::String, data::DenseTaggedImage)
+    open(filepath, "w") do io
+        write(io, data)
```

Comment:
Can you call `save` here and remove the type on data so that you don't need to add the additional `save` function?

So replace `write(io, data)` with:
```
save(Stream(format"TIFF", io, filepath), data)
```

Which'll then call L115 then L114.

Reply:
ah of course. Thanks